packages/cli/src/test-utils/ganache-test.ts

```diff
@@ -0,0 +1,63 @@
+import Web3 from 'web3'
```

Comment:
This looks familiar, is this the same as what we do in `contractkit`? If so, should we maybe create a `testing` package?

Doesn't need to be part of this PR

Reply:
yes, it's the same, it's a good follow up pr